### DISCUSSION TITLE
Is there a way to include the wp comments to the react app?

### DISCUSSION THREAD

**Discussion by ishristov** [rockiger/reactpress#26] [Ideas]
Currently when the reactpress app is displayed, only the wp header is present, but the rest of the layout are omitted, e.g. the comments sections. 

Is there a way to include the wp comments as part of the page? And maybe a more complicated question - is there a way to include the react app not as a page, but as a shortcode so that we can include is a part of another page or post?

**Comment by rockiger** [comment]
Yea, actually it is pretty simple. Just change the template of page where the React app is embedded.

No, not as a shortcode, but you can embed an app in several pages. One caveat though, you cannot use React router in the standard way. If you just have a simple app and want to use shortcodes you can use this plugin: https://wordpress.org/plugins/embed-react-application/
